types/react-native-mixpanel/react-native-mixpanel-tests.ts

```diff
@@ -0,0 +1,22 @@
+import Mixpanel = require('react-native-mixpanel');
```

Comment:
Confusingly, since you don't use `export =` syntax, `import * as Mixpanel from 'react-native-mixpanel'` works fine in this case. Of course, so does `import Mixpanel =`, but note that you can say `import { track } from 'react-native-mixpanel'`, which is not possible with modules that use `export =` syntax.

Reply:
@sandersn now I use `import * as Mixpanel from 'react-native-mixpanel'` for this file. And everything works good.